core/src/components/input/input.ionic.scss

```
@@ -1,17 +1,22 @@
 @import "./input";
 @import "./input.ionic.vars";
+@import "./input.ionic.outline.scss";
 
 // Ionic Input
 // --------------------------------------------------
+
 :host {
+  --border-width: 1px;
+  --border-color: #B8B8B8;
```

Shoot, didn't check these. My bad.

```suggestion
  --border-width: #{$ionic-border-size-small};
  --border-color: #{$ionic-color-neutral-300};
```